pkg/applicationserver/io/io.go

```diff
@@ -25,13 +25,16 @@ import (
 
 const bufferSize = 32
 
-// Server represents the Application Server to gateway frontends.
+// Server represents the Application Server to application frontends.
 type Server interface {
 	// GetBaseConfig returns the component configuration.
 	GetBaseConfig(ctx context.Context) config.ServiceBase
 	// FillContext fills the given context.
 	// This method should only be used for request contexts.
 	FillContext(ctx context.Context) context.Context
+	// SendUp sends a synthetic uplink to the Application Server, as if it came from the Network Server.
```

Comment:
It's not uplink, it's upstream traffic. The NS part in this comment can be removed.

AS is entitled to independently of NS generate upstream traffic. Uplink is indeed from devices through NS.